addons/mail/static/src/core/thread_service.js

```diff
@@ -1088,6 +1088,13 @@ export class ThreadService {
         }
         return DEFAULT_AVATAR;
     }
+
+    async notifyThreadAvatarToServer(threadId, data) {
```

Comment:
Missing type definition for params.
Especially with a param called data, it's impossible to understand what data it is.